great_expectations/render/renderer/opsgenie_renderer.py

```diff
@@ -1,14 +1,63 @@
 from __future__ import annotations
 
 import logging
+from typing import TYPE_CHECKING
 
 logger = logging.getLogger(__name__)
 
 from great_expectations.core.id_dict import BatchKwargs
 from great_expectations.render.renderer.renderer import Renderer
 
+if TYPE_CHECKING:
+    from great_expectations.checkpoint.v1_checkpoint import CheckpointResult
+    from great_expectations.core.expectation_validation_result import (
+        ExpectationSuiteValidationResult,
+    )
+
 
 class OpsgenieRenderer(Renderer):
+    def v1_render(self, checkpoint_result: CheckpointResult):
+        text_blocks: list[str] = []
+        for run_result in checkpoint_result.run_results.values():
+            text_block = self._v1_render(run_result=run_result)
+            text_blocks.append(text_block)
+
+        return self._concatenate_text_blocks(
+            checkpoint_result=checkpoint_result, text_blocks=text_blocks
+        )
+
+    def _v1_render(self, run_result: ExpectationSuiteValidationResult) -> str:
```

Comment:
This should be named to indicate it's just about a single result; first look at this method and the above made me question why one was public and one was private, and why they would be different.